addons/minio/template/base/install.sh

```diff
@@ -14,25 +14,45 @@ function minio() {
     local src="$DIR/addons/minio/__MINIO_DIR_NAME__"
     local dst="$DIR/kustomize/minio"
 
-    render_yaml_file "$src/tmpl-kustomization.yaml" > "$dst/kustomization.yaml"
-    render_yaml_file "$src/tmpl-namespace.yaml" > "$dst/namespace.yaml"
+    MINIO_HA_EXISTS=
```

Comment:
why global variable?

```suggestion
    local minio_ha_exists=
```